### ISSUE TITLE
missing hsv-colorpicker licence

### ISSUE THREAD

**Issue by FazziCLAY** [FazziCLAY/OpenToday#3]
in licence screen not found hsv-color-picker-android licence

```java

new Licence("LICENSE_hsv-alpha-color-picker-android", "hsv-alpha-color-picker-android", "https://github.com/martin-stone/hsv-alpha-color-picker-android")
```

to [OpenSourceLicenseActivity.java](https://github.com/FazziCLAY/OpenToday/commit/9dbc9693e028f6d0ff5c01e8c9a3674ea6474686#diff-599e906a2b9a9b871f907e2384636648d2c06329b8db527abd8a002fb137def6)

**Comment by FazziCLAY** [comment]
fixed in local commit. Wait...

**Comment by FazziCLAY** [comment]
# Fixed in 0.9.8 (2022.11.01)
